src/main/java/io/gravitee/resource/oauth2/generic/configuration/HttpClientOptions.java

```diff
@@ -34,6 +35,18 @@
 @AllArgsConstructor
 public class HttpClientOptions {
 
+    private static final int DEFAULT_IDLE_TIMEOUT_SECONDS = 60;
+    private static final int DEFAULT_CONNECT_TIMEOUT_MS = 10000;
+
     @Builder.Default
     private int maxConcurrentConnections = DEFAULT_MAX_CONCURRENT_CONNECTIONS;
+
+    @Builder.Default
+    private int idleTimeout = DEFAULT_IDLE_TIMEOUT_SECONDS;
+
+    @Builder.Default
+    private int connectTimeout = DEFAULT_CONNECT_TIMEOUT_MS;
+
+    @Builder.Default
+    private long requestTimeout = DEFAULT_KEEP_ALIVE_TIMEOUT;
```

Comment:
we have to respect the naming that is used here: https://github.com/gravitee-io/gravitee-api-management/blob/9706410682b54c93644a084f80d884207fccbd19/gravitee-apim-definition/gravitee-apim-definition-model/src/main/java/io/gravitee/definition/model/v4/http/HttpClientOptions.java#L60

so:

```suggestion
    private long keepAliveTimeout = DEFAULT_KEEP_ALIVE_TIMEOUT;
```